sdk/jobs/E2E-ds-experience/e2e-ml-workflow-part-I.ipynb

```diff
@@ -0,0 +1,1225 @@
+    "# Authentication package\n",
+    "from azure.identity import InteractiveBrowserCredential"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "125e8f61",
+   "metadata": {},
+   "source": [
+    "In the next cell, we enter the *Subscription ID*, *Resource Group* name and *Workspace* name. These parameters can be obtained during workspace creation, or by visiting the Azure ML Studio. The result is a handler to the workspace that we can use to manage other resources and jobs.\n",
+    "\n",
+    "Note: We use the default *interactive authentication* for this tutorial which will require you to manually confirm your connection in a browser. More advanced connection methods can be found here [sdkv1link](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.authentication?view=azure-ml-py)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "45e25c87",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get a handle to the workspace\n",
+    "ml_client = MLClient(\n",
+    "    InteractiveBrowserCredential(), \n",
+    "    subscription_id = '<SUBSCRIPTION_ID>',\n",
+    "    resource_group_name = '<RESOURCE_GROUP>',\n",
+    "    workspace_name = '<AML_WORKSPACE_NAME>'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f32afc68",
+   "metadata": {},
+   "source": [
+    "**Important**: Creating `MLClient` will not connect to the workspace. The client initialization is _lazy_, it will wait for the first time we'll need to make a call (in the notebook below, that will happen during dataset registration)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7402c1c7",
+   "metadata": {},
+   "source": [
+    "## 2.2. Register a dataset from an external url\n",
+    "\n",
+    "The data we use for our training is usually in one of the locations below:\n",
+    "- Local Machine\n",
+    "- Web\n",
+    "- Big Data Storage services (e.g. Azure Blob, Azure Data Lake Storage, SQL... )\n",
+    "\n",
+    "Azure ML uses a Dataset object to register a reusable definition of data, and consume data within a pipeline. A Dataset object is a pointer to a data storage service and a path. In the section below, we consume some data from web url as one example. Datasets from other sources can be created as well. \n",
+    "\n",
+    "For more information on various sources of data and their consumption, please check [sdkv1link](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-access-data). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d3e2d03b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ml.entities import Dataset\n",
+    "\n",
+    "web_path = \"https://archive.ics.uci.edu/ml/machine-learning-databases/00350/default%20of%20credit%20card%20clients.xls\"\n",
+    "\n",
+    "credit_dataset = Dataset(\n",
+    "    name=\"creditcard_defaults\",\n",
+    "    paths=[dict(file=web_path)],\n",
```

Comment:
Nit: `[{"file": web_path}]` makes it more obvious this is a simple dictionary.

Reply:
that is correct, however, more nested dictionaries (2,3 layers) are not very easy to eye when integrated in the cell.

Reply:
Fair point. You can keep as-is, but make sure this pain point (multiple nested dictionaries even for very simple data access) is shared with the data team.